chia/daemon/server.py

```diff
@@ -385,7 +383,6 @@ async def keyring_status(self) -> Dict[str, Any]:
         response: Dict[str, Any] = {
             "success": True,
             "is_keyring_locked": locked,
-            "passphrase_support_enabled": passphrase_support_enabled,
```

Comment:
I guess this needs to be addressed in the GUI?

Reply:
Correct. I'll create a PR for the GUI to remove this dependency.

Reply:
https://github.com/Chia-Network/chia-blockchain-gui/pull/842